app/css/openmrs-addonmanager.css

```diff
@@ -136,23 +136,31 @@ a:hover {
     padding: 0;
 }
 
-.main-home-page h2{
+.main-home-page h2 {
+    margin-left: 30px;
+}
+
+.main-home-page h2, #title-heading {
     font-size: 1.3em;
     color: #009384;
-    margin-left: 5px;
+}
+#title-heading {
+    margin-left: 45px;
+}
+
+#manage-settings {
+    margin: 0 30px;
 }
 
-.home-page-body{
+.home-page-body {
     display: flex;
     flex-direction: row;
     position: relative;
-    height: 150px;
+    /* height: 150px; */
```

Comment:
Please let us avoid commented out code.

Reply:
I forgot that, but I've already removed it. Thanks for noting that.